### PR TITLE
Add first draft for the dashboard

### PR DESCRIPTION
**Pull request by renanrodrigo**
Add sketch file for the dashboard and set up jekyll to serve a web page with the design views.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# beaker-next-design
+Design and use cases for the new beaker UI
```
Comment (StykMartin) [inline README.md]: Can we make this more descriptive? It is the first thing that a contributor can see when you open repository.

**Review by StykMartin — COMMENTED** [review]
So, we are running Ruby stack now. Is it the only available option?

**Comment by renanrodrigo** [comment]
Jekyll is the easiest way to serve static web content in github, being recommended and integrated to the page internals.
If we want, we can create a static template and serve it directly, with nothing but html+css+js, but I guess it would be extra overhead.

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+gem "just-the-docs"
+gem "github-pages", group: :jekyll_plugins
```
Comment (StykMartin) [inline Gemfile]: Replace this 
` gem "github-pages", group: :jekyll_plugins `
with
```
gem 'jekyll'
gem 'jekyll-feed'
gem 'jekyll-sitemap'
gem 'jekyll-seo-tag'
gem 'jemoji'
gem 'jekyll-readme-index'
```

**Review by StykMartin — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (renanrodrigo) [reply Gemfile]: done

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+title: Beaker-Next Design
+url: "https://beaker-project.github.io"
+remote_theme: "pmarsceill/just-the-docs"
```
Comment (StykMartin) [inline _config.yml]: theme: "just-the-docs"

**Review by StykMartin — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 3 above)
Comment (renanrodrigo) [reply _config.yml]: done

**Review by cbouchar — COMMENTED** [review]
I'm not familiar with these files so the blind question.....In .gitignore, might you need a wildcard '*' in any of these?
Not familiar with UI development; otherwise it looks good to me.

**Comment by renanrodrigo** [comment]
@StykMartin can we merge this jekyll stuff?
I could send the other screens right after that

**Pull request merged**